Formula/afl-fuzz.rb

```diff
@@ -15,7 +15,7 @@ def install
     # behaving correctly!" in a nested login shell.
     # Reported to lcamtuf@coredump.cx 6th Apr 2016.
```

Comment:
Sent an email to the mailing list asking about it, I'm not up for drafting a patch myself, as I don't understand the root cause. (Unless you just mean a patch do the same thing as this find/replace)